examples/src/main/scala/io/finch/streaming/Main.scala

```diff
@@ -18,22 +18,28 @@ import io.finch._
 *  1. `totalSum` - streaming request
 *  2. `sumTo` - streaming response
 *  3. `sumSoFar` - end-to-end (request - response) streaming
+*  4. `examples` - streaming response of examples objects
 *
 * Use the following sbt command to run the application.
 *
 * {{{
 *   $ sbt 'examples/runMain io.finch.streaming.Main'
 * }}}
+*
+* Use the following HTTPie/curl commands to test endpoints.
+*
+* {{{
+*   $ curl -X POST --header "Transfer-Encoding: chunked" -d 4 localhost:8081/totalSum
+*
+*   $ http --stream POST :8081/sumTo/3
+*
+*   $ curl -X POST --header "Transfer-Encoding: chunked" -d 4 localhost:8081/sumSoFar
+*   $ curl -X POST --header "Transfer-Encoding: chunked" -d 3 localhost:8081/sumSoFar
```

Comment:
Couldn't get those examples working with httpie if someone knows how to do it, it'd be greatly appreciated.